docs/API/v3/overview.md

```diff
@@ -0,0 +1,123 @@
+
+Resource name                                                          | Required | Description
+---------------------------------------------------------------------- | -------- | -----------
+[`PackagePublish`](package-publish-resource.md)                        | yes      | Push and delete (or unlist) packages.
+[`SearchQueryService`](search-query-service-resource.md)               | yes      | Filter and search for packages by keyword.
+[`SearchAutocompleteService`](search-autocomplete-service-resource.md) | no       | Discover package IDs and versions by substring.
+[`RegistrationsBaseUrl`](registration-base-url-resource.md)            | yes      | Get package metadata.
+[`PackageBaseAddress`](package-base-address-resource.md)               | yes      | Get package content (.nupkg).
+[`ReportAbuseUriTemplate`](report-abuse-resource.md)                   | no       | Construct a URL to access a "report abuse" web page.
+
+In general, all non-binary data returned by a V3 API resource are serialized using JSON. The response schema
+returned by each resource in the service index is defined individually for that resource. For more information about
+each resource, see the topics listed above.
+
+> [!Note]
+> When a V3 source does not implement `SearchAutocompleteService` any autocomplete behavior should be is disabled
+> gracefully. When `ReportAbuseUriTemplate` is not implemented, the official NuGet client falls back to nuget.org's
+> report abuse URL (tracked by [NuGet/Home#4924](https://github.com/NuGet/Home/issues/4924)). Other client may opt
+> to simply not show a report abuse URL to the user.
+
+## Timestamps
+
+All timestamps returned by the V3 API are UTC or are otherwise specified using ISO 8601 representation. 
+
+## HTTP methods
+
+Verb   | Use
+------ | -----------
+GET    | Performs a read-only operation, typically retrieving data.
+PUT    | Creates a resource that doesn't exist or, if it does exist, updates it. Some resources may not support update.
+DELETE | Deletes or unlists a resource.
+
+## HTTP status codes
+
+Code | Description
+---- | -----
+200  | Success, and there is a response body.
+201  | Success, and the resource was created.
+202  | Success, the request has been accepted but some work may still be incomplete and completed asynchronously.
+204  | Success, but there is no response body.
+301  | A permanent redirect.
+302  | A temporary redirect.
+400  | The parameters in the URL or in the request body aren't valid.
+401  | The provided credentials are invalid.
+403  | The action is not allowed given the provided credentials.
+404  | The requested resource doesn't exist.
+409  | The request conflicts with an existing resource.
+
+Any of the V3 API endpoints may return an HTTP redirect. Clients should gracefully handle such redirects by observing
+the `Location` header. Documentation concerning specific endpoints will not explicitly call out where redirects may be
+used.
+
+## Authentication
```

Comment:
There is only one header: `X-NuGet-ApiKey`, which is called out on the page for `PackagePublish` resource.

Reply:
There's also `X-NuGet-Client-Version`. And we have an issue tracking `X-NuGet-Session`, so it could still be useful to maintain the list from the start to keep the overview?

(btw, v2 protocol also has `NuGet-Operation`, `NuGet-DependentPackage` and `NuGet-ProjectGuids`, but those have not been ported to v3)

Reply:
Hm, makes sense. I'll give this a try.

Reply:
Fixed.